compel/src/lib/infect.c

```diff
@@ -862,11 +862,6 @@ static int compel_map_exchange(struct parasite_ctl *ctl, unsigned long size)
 	return ret;
```

Comment:
> comepl: stop rounding up the parasite
  ^coMPEL ;-)

Reply:
woops
I fixed two comepl -> compel typos
thank you